tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -849,6 +849,76 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Test the nested tabs behaviour when using container flex row @latest', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await setup( wpAdmin );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		await editor.addWidget( 'heading', container );
+
+		const tabsWidgetId = await editor.addWidget( 'nested-tabs', container );
```

Comment:
in a subsequent comment I suggest it's not necessary to store this `tabsWidgetId`. if you agree, then you can remove the setting of this `const` and just add the widget.

Reply:
Removed.